dev-games/ogre/files/ogre-1.11.1-samples.patch

```diff
@@ -0,0 +1,13 @@
+--- a/CMakeLists.txt	2018-03-23 10:48:02.309486850 +0100
++++ b/CMakeLists.txt	2018-03-23 10:48:04.157486836 +0100
+@@ -614,7 +614,9 @@
+ endif ()
+ 
+ # Setup samples
+-add_subdirectory(Samples)
++if (OGRE_BUILD_SAMPLES)
++    add_subdirectory(Samples)
```

Comment:
do you have any hopes of upstreaming your `*-samples.patch` patches? If not, you can use `cmake_comment_add_subdirectory` from cmake-utils.eclass.

Reply:
Well, I think that this is just an oversight. In all versions the Samples subdirectory is protected by OGRE_BUILD_SAMPLES.

Well, maybe it wouldn't hurt to open an upstream ticket.

Reply:
Upon further investigation it seems that the oversight is on my side. In this particular series the Samples directories own CMakeList.txt questions the OGRE_BUILD_SAMPLES variable.
...at least in 1.11.2...
So I dropped this patch as it is no longer needed.